### ISSUE TITLE
Update Grape instrumentation to support Ruby 3.0

### ISSUE THREAD

**Issue by kaylareopelle** [newrelic/newrelic-ruby-agent#821]
Grape's CI now runs on Ruby 3.0. When running the Grape multiverse suite using Ruby 3.0.2 on version 8.1.0 of the agent, lots of failures were raised. 

Tasks

- [ ]  Remove Grape limitation from Multiverse Test Runner

- [ ] Update Grape instrumentation for tests to pass on Ruby 3.0.2

- [ ] **BONUS:** Review new features on the Grape gem and add instrumentation for them

**Comment by kaylareopelle** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/APM-Agent-Overview-614b054ff9a50b000f9039d7/issues/newrelic/newrelic-ruby-agent/821?planning-poker) with Zenhub @fallwith @hannahramadan @tannalynn

**Comment by tannalynn** [comment]
Looks like this is no longer an issue. Tested on ruby 3.0, 3.1, and 3.2. Our CI actually has already been running grape, the env files has so upper ruby limits on grape versions that support 3.0+, so this hasn't been an issue for a while.
Also, I checked the changelog for grape and the releases don't appear to have any new functionality we would want to instrument, mostly bugfixes and enhancements/adjustments to existing instrumentation.
